backends/bmv2/common/JsonObjects.cpp

```diff
@@ -105,14 +104,14 @@ void JsonObjects::add_meta_info() {
     toplevel->emplace("__meta__", meta);
 }
 
-/**
- * Create a header type in json.
- * @param name header name
- * @param type header type
- * @param max_length  maximum length for a header with varbit fields;
- *                    if 0 header does not contain varbit fields
- * @param fields a JsonArray for the fields in the header
- */
+
+ /// Create a header type in json.
+ /// @param name header name
+ /// @param type header type
+ /// @param max_length  maximum length for a header with varbit fields;
+ ///                    if 0 header does not contain varbit fields
+ /// @param fields a JsonArray for the fields in the header
+
```

Comment:
You added an additional space here between the comment and the C++ code. Please remove it for all instances.